### PR TITLE
Single Window Integration Changes for SSO

### PR DESCRIPTION
**Pull request by rohit-rai-235**
this changes is regarding the single sign on.

### CODE REVIEW

**Review by vinothrallapalli-eGov — APPROVED** [review]

File: core-services/zuul/src/main/resources/routes.properties [hunk 1]
```diff
@@ -402,3 +402,10 @@ zuul.routes.bpa-services.url=http://localhost:8084/
 zuul.routes.chatbot.path=/whatsapp-webhook/**
 zuul.routes.chatbot.stripPrefix=false
 zuul.routes.chatbot.url=http://localhost:8012/
+
+
+zuul.routes.tl-services.url=/v1/external/_create
```
Comment (vinothrallapalli-eGov) [inline core-services/zuul/src/main/resources/routes.properties]: this change is not required. Remove changes in this file

**Review by vinothrallapalli-eGov — CHANGES_REQUESTED** [review]

**Comment by rohit-rai-235** [comment]
Please check i have removed this code from the route.properties

File: core-services/zuul/src/main/resources/routes.properties (hunk 1 above)
Comment (rohit-rai-235) [reply core-services/zuul/src/main/resources/routes.properties]: Sir, i have done this changes.

File: core-services/zuul/src/main/resources/application.properties [hunk 2]
```diff
@@ -23,6 +23,9 @@ egov.authorize.access.control.uri=access/v1/actions/_authorize
...
 egov.mixed-mode-endpoints-whitelist=/pgr/seva/v1/_create,/pgr/seva/v1/_search,/pgr/seva/v1/_count,/workflow/history/v1/_search,/filestore/v1/files/id,/filestore/v1/files,/filestore/v1/files/tag,/egov-common-masters/departments/_search,/wcms/masters/categorytype/_search,/wcms/masters/pipesize/_search,/wcms/masters/sourcetype/_search,/wcms/masters/supplytype/_search,/pt-property/property/propertytypes/_search,/wcms/masters/donation/_search,/wcms/masters/propertytype-categorytype/_search,/wcms/masters/propertytype-pipesize/_search,/wcms/masters/propertytype-usagetype/_search,/wcms/masters/treatmentplant/_search,/wcms-connection/connection/_getconnectiontypes,/wcms-connection/connection/_getbillingtypes,/pt-property/properties/_search,/pt-property/property/usages/_search,/egov-idgen/id/_generate,/egf-masters/financialyears/_search,/egov-common-workflows/process/_start,/egov-common-workflows/process/_search,/egov-common-workflows/tasks,/egov-common-workflows/tasks/{id}/_update,/user/_search,/user/users/_createnovalidate,/user/users/{id}/_update,/wcms-connection/connection/_create
+
+SSO_ENDPOINT=/tl-services/v1/external/_create
```
Comment (vinothrallapalli-eGov) [inline core-services/zuul/src/main/resources/application.properties:27]: remove this

File: core-services/zuul/src/main/resources/routes.properties [hunk 3]
```diff
@@ -161,7 +161,7 @@ zuul.routes.tl-masters.url=http://localhost:8084/
 
 zuul.routes.tl-services.path=/tl-services/**
 zuul.routes.tl-services.stripPrefix=false
-zuul.routes.tl-services.url=http://localhost:8084/
+zuul.routes.tl-services.url=http://localhost:8079/
```
Comment (vinothrallapalli-eGov) [inline core-services/zuul/src/main/resources/routes.properties]: revert this change

**Review by vinothrallapalli-eGov — CHANGES_REQUESTED** [review]

File: core-services/zuul/src/main/resources/routes.properties (hunk 3 above)
Comment (rohit-rai-235) [reply core-services/zuul/src/main/resources/routes.properties]: done sir

**Review by vinothrallapalli-eGov — APPROVED** [review]

**Pull request merged**